website/docs/static-assets.md

```diff
@@ -5,38 +5,41 @@ title: Static Assets
 
 In general, every website needs assets: images, stylesheets, favicons and etc. In such cases, you can create a directory named `static` at the root of your project. Every file you put into that directory will be copied into the the root of the generated `build` folder with the directory hierarchy preserved. E.g. if you add a file named `sun.jpg` to the static folder, it’ll be copied to `build/sun.jpg`.
 
-This means that if the site's `baseUrl` is `/`, an image in `/static/img/docusaurus_keytar.svg` is available at `<baseUrl>/docusaurus_keytar.svg`.
+This means that if the site's `baseUrl` is `/`, an image in `/static/img/docusaurus_keytar.svg` is available at `/img/docusaurus_keytar.svg`.
 
 <!-- TODO: Yangshun: This is inaccurate for sites with a non '/' baseUrl -->
 
 ## Referencing your static asset
 
-You can reference assets from the static folder in your code with absolute path, i.e. starting with a slash /.
+You can reference assets from the `static` folder in your code. You could use hardcoded absolute paths, i.e. starting with a slash /, but remember to include the `baseUrl` if it is not `/`. However, this will break if you change your `baseUrl` in the config.
 
-### Markdown example
+A better way would be to use the `withBaseUrl` utility function which appends the `baseUrl` to paths for you.
 
-```markdown
-<!-- reference static/img/docusaurus.png -->
+### JSX example
 
-![Docusaurus logo](/img/docusaurus.png)
 ```
+// MyComponent.js
+import withBaseUrl from '@docusaurus/withBaseUrl';
 
-Result:
+<img
+  alt="Docusaurus with Keytar"
+  src={withBaseUrl('img/docusaurus_keytar.svg')}
+/>;
+```
 
-![Docusaurus logo](/img/docusaurus.png)
+### Markdown example
```

Comment:
I believe it is possible for MDX but I couldn't `import withBaseUrl from '@docusaurus/withBaseUrl'` in MDX files. Any idea why?

Reply:
how exactly u write it ? whats the error.

Reply:
It couldn't find the imported function. I wrote it just like how I wrote JSX.

Reply:
It works now. I think I just had to restart my devserver @_@